### ISSUE TITLE
Composer installation fails for PHP 7.2

### ISSUE THREAD

**Issue by kevinvarley** [PCAPredict/magento2#27]
Composer installation fails for PHP 7.2 which is an officially supported PHP version for Magento 2.3.x

**Comment by MsEngineer** [comment]
Yes, when do you intend to make this module compatible with PHP 7.2 and Magento 2.3.x?

**Comment by danielpavey** [comment]
Hi... 

Is there any news on this?

Will this module get updated??

Thanks

**Comment by cjnewbs** [comment]
v2.0.8 released which resolves this and I'm now able to install fine.
